pkg/accumulator/resaccumulator_test.go

```diff
@@ -33,66 +33,70 @@ import (
 	"sigs.k8s.io/kustomize/pkg/types"
 )
 
-func makeResAccumulator() (*ResAccumulator, *resource.Factory, error) {
+func makeResAccumulator(t *testing.T) (*ResAccumulator, *resource.Factory, error) {
 	ra := MakeEmptyAccumulator()
 	err := ra.MergeConfig(config.MakeDefaultConfig())
 	if err != nil {
```

Comment:
Any reason not to modify this to `t.Fatalf(...)` since the other error is handled that way an this code is duplicated in every test that uses `makeResAccumulator`? You could completely remove the error return and just call Fatalf on error.

Not really that important, just code cleanup.

Reply:
good catch, done.